frontend/src/routes/Home/Search/SearchPage.tsx

```diff
@@ -152,31 +151,40 @@ function RenderSearchBar(props: {
   })
 
   useEffect(() => {
-    if (searchSchemaResults?.error || searchCompleteResults?.error) {
+    if (searchSchemaError || searchCompleteError) {
       setQueryErrors(true)
     } else {
       setQueryErrors(false)
     }
-  }, [searchSchemaResults, searchCompleteResults, queryErrors, setQueryErrors])
+  }, [searchSchemaError, searchCompleteError, queryErrors, setQueryErrors])
 
   const suggestions = useMemo(() => {
     return currentSearch === '' ||
       (!currentSearch.endsWith(':') && !operators.some((operator: string) => currentSearch.endsWith(operator)))
       ? formatSearchbarSuggestions(
-          _.get(searchSchemaResults, 'data.searchSchema.allProperties', []),
+          _.get(searchSchemaData, 'searchSchema.allProperties', [
+            'name',
+            'namespace',
+            'label',
+            'kind',
+            'cluster',
+            'apigroup',
+            'created',
+            'Loading more...',
```

Comment:
Looks like "Loading more..." was previously translated and now is not?

Reply:
Or rather "Loading..." was previously.

Reply:
Added the `Loading more...` string to translations.